packages/config/config/devices/0x031e/lzw60.json

```diff
@@ -136,19 +183,41 @@
 		},
 		"104": {
 			"label": "Battery Reporting Interval",
-			"description": "Note: The set value will round to the nearest minute.",
+			"description": "Interval, in seconds, in which battery reports should be sent. 0=Disabled. Note: The set value will round to the nearest minute.",
 			"valueSize": 4,
 			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 2678400,
-			"defaultValue": 86400,
+			"defaultValue": 7200,
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
+		"110": {
+			"label": "Send Reports According to Threshold",
```

Comment:
We're also trying to remove yes/no options. Please make this something like "Report Only When Threshold's Are Met". And make the options Enable/Disable.